prac_06/programming_language.py

```diff
@@ -0,0 +1,28 @@
+"""
+Programming language
+Estimate: 30 mins
+Actual: 60 min
+"""
+
+
+class ProgrammingLanguage:
+    """Programming class"""
+
+    def __init__(self, field="", typing="", reflection=True, year=0):
+        """Initialises language instances"""
+        self.field = field
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+
+    def __str__(self):
+        """Returns string based of programming languages"""
+        return f"{self.field}, {self.typing} Typing, Reflection={self.reflection}, First appeared in 1999"
```

Comment:
`self.year` is missing here, so all languages will seem to have appeared in 1999